book/website/reproducible-research/open/open-hardware.md

```diff
@@ -1,111 +1,359 @@
+---
+_The Turing Way_ project illustration by Scriberia. Used under a CC-BY 4.0 licence. DOI: [10.5281/zenodo.3332807](https://doi.org/10.5281/zenodo.3332807).
+```
+
+## For you
+
+People develop and share OSH for a variety of reasons. Particularly in research, OSH provides very concrete advantages:
+
+- Flexibility and Speed: you can quickly customize and combine open designs to test new research questions in an accessible way, instead of depending on vendors, their timelines and bureaucracy.
+
+- Simplicity: As OSH has a lower price tag than conventional equipment, it is much easier to obtain while bypassing the bureaucracy of contracting vendors.
+
+
+- Control: if a supplier goes out of business, users and/or third party companies have the information to keep systems running. 
+
+- Visibility: researchers and engineers developing OSH make their work more visible. They also increase opportunities for networking and impact,especially in project involving a community.
+
+- Education: By using open hardware, researchers pick new skills and fully understand how a certain tool captures data on specific phenomena/events.
+
+
+> It is worth mentioning here that building and developing OSH may require  specific knowledge and skills on one hand, and  quite a lot of time, on the other hand.
+> Buying industrial hardware with their support may be cheaper, especially if you do not have enough technical support. 
+
+
+## For science
+
+- Quality: to make science, we need research instruments; we also need to build on top of others' knowledge. A contributor might offer a totally novel solution to a design problem that would never occur to you; contributors may also catch errors that you’ve missed. 
+
+- Reproducibility: OSH designs can be replicated, allowing for verification and reproduction of experiments and data. Moreover, users can have much better control on the calibration of their devices, boosting replicability even further.
+
+- Inclusiveness: OSH can make science possible where resources are scarce. Local adaptation and production of OSH helps reducing the impact of import restrictions and bureaucracies that obstaculise science. Also, more students may be able to interact with scientific tools in hands-on sessions, as the equipment is cheaper.
+
+- Sustainability: OSH means you have all the information for repairing and maintaining devices locally, extending the lifespan of the product and reducing waste. 
+
+
+Many researchers who develop or use OSH for research also value OSH as an educational tool.
+Working with OSH designs allows students and researchers to fully understand how a certain tool captures data on specific phenomena/events.
+
+
+
+
+
+# How to make hardware open source
+
+
+In order to make your hardware reusable and modifyable by others, its source should be shared with an appropriate license.
+This is usually done via specific online platforms.
```

Comment:
can you share some examples here of the platforms or even specific examples of shared OSH project on these platforms?